File: src/main/java/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/InlineQueryResultCachedVoice.java
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult;

import com.fasterxml.jackson.annotation.JsonProperty;

/**
 * @see <a href="https://core.telegram.org/bots/api#inlinequeryresultcachedvoice">InlineQueryResultCachedVoice</a>
 */
public class InlineQueryResultCachedVoice extends TitledInlineResult {
    private static final String TYPE = "voice";

    /**
     * voice_file_id String A valid file identifier for the voice message
     */
    @JsonProperty("voice_file_id")
    private String voiceFileId = null;

    InlineQueryResultCachedVoice() {
        super(TYPE);
    }

    public String getVoiceFileId() {
        return voiceFileId;
    }

    public void setVoiceFileId(String voiceFileId) {
        this.voiceFileId = voiceFileId;
    }
}

File: src/main/kotlin/io/github/oleksivio/telegram/bot/api/model/objects/inline/queryresult/InlineQueryResultCachedVoice.kt
package io.github.oleksivio.telegram.bot.api.model.objects.inline.queryresult

import com.fasterxml.jackson.annotation.JsonProperty

/**
 * @see [InlineQueryResultCachedVoice](https://core.telegram.org/bots/api/#inlinequeryresultcachedvoice)
 */
class InlineQueryResultCachedVoice(
        /**
         * voice_file_id String A valid file identifier for the voice message
         */
        @JsonProperty("voice_file_id")
        var voiceFileId: String,
        /**
         * type String Type of the result
         */
        @JsonProperty("type")
        val type: String = "voice"
) : TitledInlineResult()
